opencensus/copts.bzl

```diff
@@ -33,7 +33,7 @@ load(
 
 DEFAULT_COPTS = select({
     "//opencensus:llvm_compiler": LLVM_FLAGS,
-    "//opencensus:windows": MSVC_FLAGS,
+    "//opencensus:windows": MSVC_FLAGS + ["/wd4715"],
```

Comment:
Please add a comment about what 4715 is.

Also: not needed in TEST_COPTS?

Reply:
Commented ("not all control paths return a value", treated as an error). I added it to TEST_COPTS for good measure--I do not think any of our code exercises that warning.